src/webui/src/components/public-child/OpenRow.tsx

```diff
@@ -58,7 +58,7 @@ class OpenRow extends React.Component<OpenRowProps, {}> {
                         <TabPane tab="Log" key="2">
                             {
                                 trainingPlatform === 'pai' || trainingPlatform === 'kubeflow' ||
-                                trainingPlatform === 'frameworkcontroller'
+                                trainingPlatform === 'frameworkcontroller' || trainingPlatform === 'remote'
```

Comment:
The same as above. Suggest to change to : 
trainingPlatform !== 'local'

Reply:
changed to `trainingPlatform !== 'local'`